cli/asc.js

```diff
@@ -193,8 +193,9 @@ exports.main = function main(argv, options, callback) {
   if (!stdout) throw Error("'options.stdout' must be specified");
   if (!stderr) throw Error("'options.stderr' must be specified");
 
-  const opts = optionsUtil.parse(argv, exports.options);
-  let args = opts.options;
+  const opts = optionsUtil.parse(argv, exports.options, false);
+  let args = optionsUtil.addDefaults(exports.options, opts.options);
```

Comment:
This is happening too early, and is instead intended to be performed as the last step once CLI args and asconfigs have been fully processed. For that, we just need to apply the defaults for `target` and `config` early, independently of populating their values. Can either hardcode these or pull them from the config.

Reply:
I changed addDefaults so that it returns a new object.  This way `opts.options` is untouched.  Then at the end of the config it is merged back into args.  It was the only way I found that fixed the issues your having in #1406.

Reply:
Hmm, I suspect that the underlying issue is that the relative path transform is applied to imported file names, in turn stripping the `.` from `./a`, resulting in the packages test trying to find `a`, which resolves differently. While the changes in ordering here apparently prevent this, somehow, that doesn't look like a proper fix.